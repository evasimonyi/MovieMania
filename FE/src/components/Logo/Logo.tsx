const Logo = () => {
  return (
    <span>
      <strong>Movie</strong>Mania
    </span>
  );
};

export default Logo;
