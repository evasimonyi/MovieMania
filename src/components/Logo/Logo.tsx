import React from 'react';

const Logo = () => {
  return (
    <span className='text-pastel-grey m-4'>
      <strong>Movie</strong>Mania
    </span>
  )
}

export default Logo;
