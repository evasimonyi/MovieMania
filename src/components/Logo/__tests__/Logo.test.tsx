import React from 'react';
import { render, screen } from '@testing-library/react';
import Logo from '../Logo';

test('Logo renders well', () => {
  render(<Logo />)

  expect(screen.getByText(/netflix/i)).toBeInTheDocument();
  expect(screen.getByText(/roulette/i)).toBeInTheDocument();
})

test('Logo snapshot', () => {
  const { container } = render(<Logo />);
  expect(container.firstChild).toMatchSnapshot();
})