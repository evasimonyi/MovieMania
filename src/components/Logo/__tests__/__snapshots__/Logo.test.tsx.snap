// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo snapshot 1`] = `
<span>
  <strong>
    netflix
  </strong>
  roulette
</span>
`;
